android.content.res.Resources$NotFoundException: Unable to find resource ID #0xffffffff

### NOTES

The issue here appears to be that google removed some resources/apis that we are compiling against.  It looks like we will need to start pinning the AndroidX dlls or compile against newer releases of AndroidX at a faster cadence 

### Original ticket


I updated my project this morning with some new android files via newget and ever since, when I try to navigate using the flyout menu, I get the following error:  android.content.res.Resources$NotFoundException: Unable to find resource ID #0xffffffff

### Description
The files I updated that caused this issue were:

Xamarin.AndroidX.MediaRouter 1.2.1 -> 1.2.2.1
Xamarin.AndroidX.LeanBack 1.0.0.6 -> 1.0.0.7
Xamarin.AndroidX.Palette 1.0.0.6 -> 1.0.0.7

when I rolled the files back I no longer have the issue


### Steps to Reproduce

1. Update your Xamarin app to the latest android files
2. Try to use flyout navigation 
3.  I know one of the files I updated was Xamarin.AndroidX.Palette

### Expected Behavior

### Actual Behavior

### Basic Information

- Version with issue:
- Last known good version:
- Platform Target Frameworks: <!-- All that apply -->
  - iOS:  <!-- The version of the iOS SDK you are compiling against, e.g. 11.1 -->
  - Android: <!-- The version of the Android SDK you are compiling against, e.g. 7.1 --> 
  - UWP:  <!-- The version of the UWP SDK you are compiling against, e.g. 16299 --> 
- Android Support Library / AndroidX Version: <!-- if applicable -->
- NuGet Packages:
- Affected Devices:

### Environment
Microsoft Visual Studio Professional 2019 Version 16.8.6
Xamarin Forms 5.0.0.2012
<!--
1.
Visual Studio: Help > About Microsoft Visual Studio > Copy Info [button]
Visual Studio for Mac: Visual Studio > About Visual Studio > Show Details > Copy Information [button]
2. Paste into the code block below (between ```)
-->

<details><summary>Show/Hide Visual Studio info</summary>

```
```

</details>

### Build Logs

<!--
If your issue is related to a build error, be sure to attach build logs.

Binary logs are preferred instead of text logs.  Please save and attach a .binlog if possible:
(https://docs.microsoft.com/en-us/visualstudio/msbuild/obtaining-build-logs-with-msbuild?view=vs-2019#save-a-binary-log)

Otherwise, a text log is still helpful:

1. Place cursor below this comment block.
2. Attach build log or link to gist (https://gist.github.com/) of the log.
3. Logs can be found in the Build Output tab of the Errors pad in Visual Studio
To get full build logs, set the log verbosity to diagnostic at the following locations:
- On Visual Studio for Mac: Preferences > Projects > Build
- On Visual Studio for Windows: Tools > Options > Projects and Solutions > Build and Run
On Visual Studio for Windows you also want to add `-v -v -v -v` to the mtouch additional arguments by right-clicking the project in the solution explorer and selecting `Properties`.
-->


### Screenshots

<!-- If the issue is a visual issue, please include screenshots showing the problem if possible -->

### Reproduction Link

<!-- Please upload or provide a link to a reproduction case -->

### Workaround

<!-- Did you find any workaround for this issue? This can unblock other people while waiting for this issue to be resolved -->

I'm having this same problem, if I downgrade Xamarin.AndroidX.AppCompat to 1.2.0.6, so that I can use  Xamarin.AndroidX.Fragment 1.2.5.4 then it won't compile. If  Xamarin.AndroidX.Fragment (version 1.3.1) still crashes my app runs but it will still fail when changing pages in the Flyout of the Shell.

This are the errors when using AppCompat lower that 1.2.0.7:

obj\Release\110\android\src\mono\androidx\fragment\app\FragmentOnAttachListenerImplementor.java(8,24): javac.exe error JAVAC0000:  error: cannot find symbol
obj\Release\110\android\src\mono\androidx\fragment\app\FragmentOnAttachListenerImplementor.java(8,24): javac.exe error JAVAC0000: 		androidx.fragment.app.FragmentOnAttachListener
ojb\Release\110\android\src\mono\androidx\fragment\app\FragmentOnAttachListenerImplementor.java(8,24): javac.exe error JAVAC0000:   symbol:   class FragmentOnAttachListener
obj\Release\110\android\src\mono\androidx\fragment\app\FragmentOnAttachListenerImplementor.java(8,24): javac.exe error JAVAC0000:   location: package androidx.fragment.app
obj\Release\110\android\src\mono\androidx\fragment\app\FragmentOnAttachListenerImplementor.java(8,24): javac.exe error JAVAC0000: 
obj\Release\110\android\src\mono\androidx\fragment\app\FragmentResultListenerImplementor.java(8,24): javac.exe error JAVAC0000:  error: cannot find symbol
obj\Release\110\android\src\mono\androidx\fragment\app\FragmentResultListenerImplementor.java(8,24): javac.exe error JAVAC0000: 		androidx.fragment.app.FragmentResultListener
obj\Release\110\android\src\mono\androidx\fragment\app\FragmentResultListenerImplementor.java(8,24): javac.exe error JAVAC0000:   symbol:   class FragmentResultListener
obj\Release\110\android\src\mono\androidx\fragment\app\FragmentResultListenerImplementor.java(8,24): javac.exe error JAVAC0000:   location: package androidx.fragment.app
obj\Release\110\android\src\mono\androidx\fragment\app\FragmentResultListenerImplementor.java(8,24): javac.exe error JAVAC0000:

--- a/.nuspec/Xamarin.Forms.AppLinks.nuspec
+++ b/.nuspec/Xamarin.Forms.AppLinks.nuspec
@@ -17,7 +17,7 @@
       <group targetFramework="MonoAndroid10.0">
         <dependency id="Xamarin.Firebase.AppIndexing" version="119.1.0"/>        
         <dependency id="Xamarin.AndroidX.Lifecycle.LiveData" version="2.2.0.4"/>
-        <dependency id="Xamarin.AndroidX.Legacy.Support.V4" version="1.0.0.6"/>
+        <dependency id="Xamarin.AndroidX.Legacy.Support.V4" version="1.0.0.7"/>
       </group>
     </dependencies>
     <references>

--- a/.nuspec/Xamarin.Forms.Visual.Material.nuspec
+++ b/.nuspec/Xamarin.Forms.Visual.Material.nuspec
@@ -22,8 +22,8 @@
       </group>
 
       <group targetFramework="MonoAndroid10.0">
-        <dependency id="Xamarin.AndroidX.Lifecycle.LiveData" version="2.2.0.4" />
-        <dependency id="Xamarin.Google.Android.Material" version="1.2.1.1" />
+        <dependency id="Xamarin.AndroidX.Lifecycle.LiveData" version="[2.2.*]" />
+        <dependency id="Xamarin.Google.Android.Material" version="[1.2.*]" />
       </group>
       <group targetFramework="Xamarin.iOS10">
         <dependency id="Xamarin.iOS.MaterialComponents" version="92.0.0"/>

--- a/.nuspec/Xamarin.Forms.Visual.Material.nuspec
+++ b/.nuspec/Xamarin.Forms.Visual.Material.nuspec
@@ -22,8 +22,8 @@
       </group>
 
       <group targetFramework="MonoAndroid10.0">
-        <dependency id="Xamarin.AndroidX.Lifecycle.LiveData" version="[2.2.*]" />
-        <dependency id="Xamarin.Google.Android.Material" version="[1.2.*]" />
+        <dependency id="Xamarin.AndroidX.Lifecycle.LiveData" version="[2.2.0.4,2.3)" />
+        <dependency id="Xamarin.Google.Android.Material" version="[1.2.1.1,1.3)" />
       </group>
       <group targetFramework="Xamarin.iOS10">
         <dependency id="Xamarin.iOS.MaterialComponents" version="92.0.0"/>

--- a/.nuspec/Xamarin.Forms.nuspec
+++ b/.nuspec/Xamarin.Forms.nuspec
@@ -15,17 +15,17 @@
     <copyright>© Microsoft Corporation. All rights reserved.</copyright>
     <dependencies>
       <group targetFramework="MonoAndroid10.0">
-        <dependency id="Xamarin.AndroidX.Lifecycle.LiveData" version="2.2.0.4" />
-        <dependency id="Xamarin.Google.Android.Material" version="1.2.1.1" />
-        <dependency id="Xamarin.AndroidX.Legacy.Support.V4" version="1.0.0.7" />
-        <dependency id="Xamarin.AndroidX.Browser" version="1.3.0" />
+        <dependency id="Xamarin.AndroidX.Lifecycle.LiveData" version="[2.2.*]" />
+        <dependency id="Xamarin.Google.Android.Material" version="[1.2.*]" />
+        <dependency id="Xamarin.AndroidX.Legacy.Support.V4" version="[1.0.*]" />
+        <dependency id="Xamarin.AndroidX.Browser" version="[1.3.*]" />
 
         <!-- Because of a misspelled reference these all need to be explicitly included so the fixed version 
           is transitively loaded  https://github.com/xamarin/AndroidX/issues/283#issuecomment-788103573 -->
-        <dependency id="Xamarin.AndroidX.Core" version="1.3.2.3" />
-        <dependency id="Xamarin.AndroidX.CustomView" version="1.1.0.6" />
-        <dependency id="Xamarin.AndroidX.Preference" version="1.1.1.8" />
-        <dependency id="Xamarin.AndroidX.RecyclerView" version="1.1.0.8" />
+        <dependency id="Xamarin.AndroidX.Core" version="[1.3.*]" />
+        <dependency id="Xamarin.AndroidX.CustomView" version="[1.1.*]" />
+        <dependency id="Xamarin.AndroidX.Preference" version="[1.1.*]" />
+        <dependency id="Xamarin.AndroidX.RecyclerView" version="[1.1.*]" />
       </group>
       <group targetFramework="uap10.0.14393">
         <dependency id="NETStandard.Library" version="2.0.1"/>

--- a/.nuspec/Xamarin.Forms.nuspec
+++ b/.nuspec/Xamarin.Forms.nuspec
@@ -17,8 +17,15 @@
       <group targetFramework="MonoAndroid10.0">
         <dependency id="Xamarin.AndroidX.Lifecycle.LiveData" version="2.2.0.4" />
         <dependency id="Xamarin.Google.Android.Material" version="1.2.1.1" />
-        <dependency id="Xamarin.AndroidX.Legacy.Support.V4" version="1.0.0.6" />
+        <dependency id="Xamarin.AndroidX.Legacy.Support.V4" version="1.0.0.7" />
         <dependency id="Xamarin.AndroidX.Browser" version="1.3.0" />
+
+        <!-- Because of a misspelled reference these all need to be explicitly included so the fixed version 
+          is transitively loaded  https://github.com/xamarin/AndroidX/issues/283#issuecomment-788103573 -->
+        <dependency id="Xamarin.AndroidX.Core" version="1.3.2.3" />
+        <dependency id="Xamarin.AndroidX.CustomView" version="1.1.0.6" />
+        <dependency id="Xamarin.AndroidX.Preference" version="1.1.1.8" />
+        <dependency id="Xamarin.AndroidX.RecyclerView" version="1.1.0.8" />
       </group>
       <group targetFramework="uap10.0.14393">
         <dependency id="NETStandard.Library" version="2.0.1"/>

--- a/.nuspec/Xamarin.Forms.nuspec
+++ b/.nuspec/Xamarin.Forms.nuspec
@@ -15,17 +15,17 @@
     <copyright>© Microsoft Corporation. All rights reserved.</copyright>
     <dependencies>
       <group targetFramework="MonoAndroid10.0">
-        <dependency id="Xamarin.AndroidX.Lifecycle.LiveData" version="[2.2.*]" />
-        <dependency id="Xamarin.Google.Android.Material" version="[1.2.*]" />
-        <dependency id="Xamarin.AndroidX.Legacy.Support.V4" version="[1.0.*]" />
-        <dependency id="Xamarin.AndroidX.Browser" version="[1.3.*]" />
+        <dependency id="Xamarin.AndroidX.Lifecycle.LiveData" version="[2.2.0.4,2.3)" />
+        <dependency id="Xamarin.Google.Android.Material" version="[1.2.1.1,1.3)" />
+        <dependency id="Xamarin.AndroidX.Legacy.Support.V4" version="[1.0.0.7,1.1)" />
+        <dependency id="Xamarin.AndroidX.Browser" version="[1.3.0,1.4)" />
 
         <!-- Because of a misspelled reference these all need to be explicitly included so the fixed version 
           is transitively loaded  https://github.com/xamarin/AndroidX/issues/283#issuecomment-788103573 -->
-        <dependency id="Xamarin.AndroidX.Core" version="[1.3.*]" />
-        <dependency id="Xamarin.AndroidX.CustomView" version="[1.1.*]" />
-        <dependency id="Xamarin.AndroidX.Preference" version="[1.1.*]" />
-        <dependency id="Xamarin.AndroidX.RecyclerView" version="[1.1.*]" />
+        <dependency id="Xamarin.AndroidX.Core" version="[1.3.2.3,1.4)" />
+        <dependency id="Xamarin.AndroidX.CustomView" version="[1.1.0.6,1.2)" />
+        <dependency id="Xamarin.AndroidX.Preference" version="[1.1.1.8,1.2)" />
+        <dependency id="Xamarin.AndroidX.RecyclerView" version="[1.1.0.8,1.2)" />
       </group>
       <group targetFramework="uap10.0.14393">
         <dependency id="NETStandard.Library" version="2.0.1"/>

--- a/.nuspec/Xamarin.Forms.nuspec
+++ b/.nuspec/Xamarin.Forms.nuspec
@@ -18,7 +18,7 @@
         <dependency id="Xamarin.AndroidX.Lifecycle.LiveData" version="[2.2.0.4,2.3)" />
         <dependency id="Xamarin.Google.Android.Material" version="[1.2.1.1,1.3)" />
         <dependency id="Xamarin.AndroidX.Legacy.Support.V4" version="[1.0.0.7,1.1)" />
-        <dependency id="Xamarin.AndroidX.Browser" version="[1.3.0,1.4)" />
+        <dependency id="Xamarin.AndroidX.Browser" version="[1.3.0.5,1.4)" />
 
         <!-- Because of a misspelled reference these all need to be explicitly included so the fixed version 
           is transitively loaded  https://github.com/xamarin/AndroidX/issues/283#issuecomment-788103573 -->
@@ -26,7 +26,7 @@
         <dependency id="Xamarin.AndroidX.CustomView" version="[1.1.0.6,1.2)" />
         <dependency id="Xamarin.AndroidX.Preference" version="[1.1.1.8,1.2)" />
         <dependency id="Xamarin.AndroidX.RecyclerView" version="[1.1.0.8,1.2)" />
-        <dependency id="Xamarin.AndroidX.Navigation.UI" version="2.3.2.1" />
+        <dependency id="Xamarin.AndroidX.Navigation.UI" version="[2.3.2.1,2.4)" />
       </group>
       <group targetFramework="uap10.0.14393">
         <dependency id="NETStandard.Library" version="2.0.1"/>

--- a/DualScreen/DualScreen.Android/DualScreen.Android.csproj
+++ b/DualScreen/DualScreen.Android/DualScreen.Android.csproj
@@ -115,7 +115,9 @@
     </ProjectReference>
   </ItemGroup>
   <ItemGroup>
-    <PackageReference Include="Xamarin.AndroidX.Legacy.Support.V4" Version="1.0.0.7" />
+    <PackageReference Include="Xamarin.AndroidX.Legacy.Support.V4" Version="1.0.0.7" />    
+    <PackageReference Include="Xamarin.AndroidX.Lifecycle.LiveData" Version="2.2.0.4" />
+    <PackageReference Include="Xamarin.Google.Android.Material" Version="1.2.1.1" />
   </ItemGroup>
   <ItemGroup>
     <AndroidResource Include="Resources\values\colors.xml" />

--- a/DualScreen/DualScreen.Android/DualScreen.Android.csproj
+++ b/DualScreen/DualScreen.Android/DualScreen.Android.csproj
@@ -115,9 +115,7 @@
     </ProjectReference>
   </ItemGroup>
   <ItemGroup>
-    <PackageReference Include="Xamarin.AndroidX.Lifecycle.LiveData" Version="2.2.0.4" />
-    <PackageReference Include="Xamarin.Google.Android.Material" Version="1.2.1.1" />
-    <PackageReference Include="Xamarin.AndroidX.Legacy.Support.V4" Version="1.0.0.6" />
+    <PackageReference Include="Xamarin.AndroidX.Legacy.Support.V4" Version="1.0.0.7" />
   </ItemGroup>
   <ItemGroup>
     <AndroidResource Include="Resources\values\colors.xml" />

--- a/EmbeddingTestBeds/Embedding.Droid/Directory.Build.targets
+++ b/EmbeddingTestBeds/Embedding.Droid/Directory.Build.targets
@@ -4,6 +4,6 @@
     <PackageReference Include="Xamarin.AndroidX.Palette" Version="1.0.0.1" />
     <PackageReference Include="Xamarin.AndroidX.Lifecycle.LiveData" Version="2.1.0.1" />
     <PackageReference Include="Xamarin.Google.Android.Material" Version="1.2.1.1" />
-    <PackageReference Include="Xamarin.AndroidX.Legacy.Support.V4" Version="1.0.0.1" />
+    <PackageReference Include="Xamarin.AndroidX.Legacy.Support.V4" Version="1.0.0.7" />
   </ItemGroup>
 </Project>

--- a/EmbeddingTestBeds/Embedding.Droid/Directory.Build.targets
+++ b/EmbeddingTestBeds/Embedding.Droid/Directory.Build.targets
@@ -1,6 +1,6 @@
 <Project>
   <ItemGroup>
-    <PackageReference Include="Xamarin.AndroidX.Browser" Version="1.3.0" />
+    <PackageReference Include="Xamarin.AndroidX.Browser" Version="1.3.0.5" />
     <PackageReference Include="Xamarin.AndroidX.Palette" Version="1.0.0.1" />
     <PackageReference Include="Xamarin.AndroidX.Lifecycle.LiveData" Version="2.1.0.1" />
     <PackageReference Include="Xamarin.Google.Android.Material" Version="1.2.1.1" />

--- a/Stubs/Xamarin.Forms.Platform.Android/Xamarin.Forms.Platform.Android (Forwarders).csproj
+++ b/Stubs/Xamarin.Forms.Platform.Android/Xamarin.Forms.Platform.Android (Forwarders).csproj
@@ -23,8 +23,8 @@
   </ItemGroup>
   <ItemGroup Condition="'$(TargetFramework)' == 'MonoAndroid10.0'">
     <PackageReference Include="Xamarin.AndroidX.Lifecycle.LiveData" Version="2.2.0.4" />
-    <PackageReference Include="Xamarin.AndroidX.Legacy.Support.V4" Version="1.0.0.6" />
-    <PackageReference Include="Xamarin.AndroidX.RecyclerView" Version="1.1.0.6" />
+    <PackageReference Include="Xamarin.AndroidX.Legacy.Support.V4" Version="1.0.0.7" />
+    <PackageReference Include="Xamarin.AndroidX.RecyclerView" Version="1.1.0.8" />
     <PackageReference Include="Xamarin.Google.Android.Material" Version="1.2.1.1" />
   </ItemGroup>
 </Project>

--- a/Xamarin.Forms.ControlGallery.Android/ContentDescriptionEffectRenderer.cs
+++ b/Xamarin.Forms.ControlGallery.Android/ContentDescriptionEffectRenderer.cs
@@ -4,8 +4,8 @@ using Xamarin.Forms;
 using Xamarin.Forms.ControlGallery.Android;
 using Xamarin.Forms.Controls.Issues;
 using Xamarin.Forms.Platform.Android;
-using AndroidX.Core.View.Accessibiity;
 using AndroidX.Core.View;
+using AndroidX.Core.View.Accessibility;
 
 [assembly: ExportEffect(typeof(ContentDescriptionEffectRenderer), ContentDescriptionEffect.EffectName)]
 namespace Xamarin.Forms.ControlGallery.Android

--- a/Xamarin.Forms.ControlGallery.Android/Directory.Build.targets
+++ b/Xamarin.Forms.ControlGallery.Android/Directory.Build.targets
@@ -1,7 +1,7 @@
 <Project>
   <ItemGroup>
     <PackageReference Include="Xamarin.AndroidX.Lifecycle.LiveData" Version="2.2.0.4" />
-    <PackageReference Include="Xamarin.AndroidX.Browser" Version="1.3.0" />
+    <PackageReference Include="Xamarin.AndroidX.Browser" Version="1.3.0.5" />
     <PackageReference Include="Xamarin.Google.Android.Material" Version="1.2.1.1" />
     <PackageReference Include="Xamarin.AndroidX.Legacy.Support.V4" Version="1.0.0.7" />
     <PackageReference Include="Xamarin.AndroidX.Palette" Version="1.0.0.6" />

--- a/Xamarin.Forms.ControlGallery.Android/Directory.Build.targets
+++ b/Xamarin.Forms.ControlGallery.Android/Directory.Build.targets
@@ -3,7 +3,7 @@
     <PackageReference Include="Xamarin.AndroidX.Lifecycle.LiveData" Version="2.2.0.4" />
     <PackageReference Include="Xamarin.AndroidX.Browser" Version="1.3.0" />
     <PackageReference Include="Xamarin.Google.Android.Material" Version="1.2.1.1" />
-    <PackageReference Include="Xamarin.AndroidX.Legacy.Support.V4" Version="1.0.0.6" />
+    <PackageReference Include="Xamarin.AndroidX.Legacy.Support.V4" Version="1.0.0.7" />
     <PackageReference Include="Xamarin.AndroidX.Palette" Version="1.0.0.6" />
   </ItemGroup>
   <Import Project="Nuget.Build.targets" />

--- a/Xamarin.Forms.ControlGallery.Android/Nuget.Build.targets
+++ b/Xamarin.Forms.ControlGallery.Android/Nuget.Build.targets
@@ -8,7 +8,7 @@
         <PackageReference Include="Xamarin.AndroidX.Lifecycle.LiveData" Version="2.1.0" />
         <PackageReference Include="Xamarin.AndroidX.Browser" Version="1.3.0" />
         <PackageReference Include="Xamarin.Google.Android.Material" Version="1.2.1.1" />
-        <PackageReference Include="Xamarin.AndroidX.Legacy.Support.V4" Version="1.0.0" />
+        <PackageReference Include="Xamarin.AndroidX.Legacy.Support.V4" Version="1.0.0.7" />
         <PackageReference Include="Xamarin.AndroidX.Palette" Version="1.0.0" />
     </ItemGroup>
 </Project>

--- a/Xamarin.Forms.ControlGallery.Android/Nuget.Build.targets
+++ b/Xamarin.Forms.ControlGallery.Android/Nuget.Build.targets
@@ -6,7 +6,7 @@
     </ItemGroup>
     <ItemGroup>
         <PackageReference Include="Xamarin.AndroidX.Lifecycle.LiveData" Version="2.1.0" />
-        <PackageReference Include="Xamarin.AndroidX.Browser" Version="1.3.0" />
+        <PackageReference Include="Xamarin.AndroidX.Browser" Version="1.3.0.5" />
         <PackageReference Include="Xamarin.Google.Android.Material" Version="1.2.1.1" />
         <PackageReference Include="Xamarin.AndroidX.Legacy.Support.V4" Version="1.0.0.7" />
         <PackageReference Include="Xamarin.AndroidX.Palette" Version="1.0.0" />

--- a/Xamarin.Forms.ControlGallery.Android/Xamarin.Forms.ControlGallery.Android.csproj
+++ b/Xamarin.Forms.ControlGallery.Android/Xamarin.Forms.ControlGallery.Android.csproj
@@ -350,6 +350,11 @@
     <PackageReference Include="Xam.Plugin.DeviceInfo">
       <Version>3.0.2</Version>
     </PackageReference>
+    <PackageReference Include="Xamarin.AndroidX.Legacy.Support.V4" Version="1.0.0.7" />
+    <PackageReference Include="Xamarin.AndroidX.Core" Version="1.3.2.3" />
+    <PackageReference Include="Xamarin.AndroidX.CustomView" Version="1.1.0.6" />
+    <PackageReference Include="Xamarin.AndroidX.Preference" Version="1.1.1.8" />
+    <PackageReference Include="Xamarin.AndroidX.RecyclerView" Version="1.1.0.8" />
   </ItemGroup>
   <ItemGroup>
     <AndroidAsset Include="Assets\fonts\ionicons.ttf">

--- a/Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/Issue6878.cs
+++ b/Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/Issue6878.cs
@@ -69,6 +69,10 @@ namespace Xamarin.Forms.Controls.Issues
 				{
 					Items[0].Items.Clear();
 					Items.Clear();
+
+					ContentPage parent = _stackContent.Parent as ContentPage;
+					parent.Content = null;
+
 					AddTopTab(TopTab).Content = _stackContent;
 					CurrentItem = Items.Last();
 

--- a/Xamarin.Forms.DualScreen/Xamarin.Forms.DualScreen.csproj
+++ b/Xamarin.Forms.DualScreen/Xamarin.Forms.DualScreen.csproj
@@ -57,6 +57,6 @@
   <ItemGroup Condition="'$(TargetFramework)' == 'MonoAndroid10.0'">
     <PackageReference Include="Xamarin.AndroidX.Lifecycle.LiveData" Version="2.2.0.4" />
     <PackageReference Include="Xamarin.Google.Android.Material" Version="1.2.1.1" />
-    <PackageReference Include="Xamarin.AndroidX.Legacy.Support.V4" Version="1.0.0.6" />
+    <PackageReference Include="Xamarin.AndroidX.Legacy.Support.V4" Version="1.0.0.7" />
   </ItemGroup>
 </Project>

--- a/Xamarin.Forms.Maps.Android/Xamarin.Forms.Maps.Android.csproj
+++ b/Xamarin.Forms.Maps.Android/Xamarin.Forms.Maps.Android.csproj
@@ -22,7 +22,7 @@
   <ItemGroup>
     <PackageReference Include="Xamarin.AndroidX.Lifecycle.LiveData" Version="2.2.0.4" />
     <PackageReference Include="Xamarin.Google.Android.Material" Version="1.2.1.1" />
-    <PackageReference Include="Xamarin.AndroidX.Legacy.Support.V4" Version="1.0.0.6" />
+    <PackageReference Include="Xamarin.AndroidX.Legacy.Support.V4" Version="1.0.0.7" />
     <PackageReference Include="Xamarin.GooglePlayServices.Maps" Version="117.0.0" />
   </ItemGroup>
 </Project>

--- a/Xamarin.Forms.Platform.Android.AppLinks/Xamarin.Forms.Platform.Android.AppLinks.csproj
+++ b/Xamarin.Forms.Platform.Android.AppLinks/Xamarin.Forms.Platform.Android.AppLinks.csproj
@@ -26,6 +26,6 @@
   </ItemGroup>
   <ItemGroup Condition="'$(TargetFramework)' == 'MonoAndroid10.0'">
     <PackageReference Include="Xamarin.AndroidX.Lifecycle.LiveData" Version="2.2.0.4" />
-    <PackageReference Include="Xamarin.AndroidX.Legacy.Support.V4" Version="1.0.0.6" />
+    <PackageReference Include="Xamarin.AndroidX.Legacy.Support.V4" Version="1.0.0.7" />
   </ItemGroup>
 </Project>

--- a/Xamarin.Forms.Platform.Android/AccessibilityDelegateAutomationId.cs
+++ b/Xamarin.Forms.Platform.Android/AccessibilityDelegateAutomationId.cs
@@ -4,7 +4,7 @@ using System.Text;
 using Android.Views.Accessibility;
 using AndroidX.AppCompat.Widget;
 using AndroidX.Core.View;
-using AndroidX.Core.View.Accessibiity;
+using AndroidX.Core.View.Accessibility;
 using Xamarin.Forms.Platform.Android.FastRenderers;
 
 namespace Xamarin.Forms.Platform.Android

--- a/Xamarin.Forms.Platform.Android/AppCompat/FormsAppCompatActivity.cs
+++ b/Xamarin.Forms.Platform.Android/AppCompat/FormsAppCompatActivity.cs
@@ -223,6 +223,7 @@ namespace Xamarin.Forms.Platform.Android
 				// because we're rebuilding everything from scratch. This saves a bit of memory
 				// and prevents loading errors from child fragment managers
 				savedInstanceState?.Remove("android:support:fragments");
+				savedInstanceState?.Remove("androidx.lifecycle.BundlableSavedStateRegistry.key");
 			}
 
 			Profile.FramePartition("Xamarin.Android.OnCreate");

--- a/Xamarin.Forms.Platform.Android/AppCompat/FormsFragmentPagerAdapter.cs
+++ b/Xamarin.Forms.Platform.Android/AppCompat/FormsFragmentPagerAdapter.cs
@@ -7,14 +7,18 @@ using FragmentTransit = Android.App.FragmentTransit;
 
 namespace Xamarin.Forms.Platform.Android.AppCompat
 {
+#pragma warning disable CS0618 // Type or member is obsolete
 	internal class FormsFragmentPagerAdapter<T> : FragmentPagerAdapter where T : Page
+#pragma warning restore CS0618 // Type or member is obsolete
 	{
 		MultiPage<T> _page;
 		FragmentManager _fragmentManager;
 		List<Fragment> _fragments;
 		bool _disposed;
 
+#pragma warning disable CS0618 // Type or member is obsolete
 		public FormsFragmentPagerAdapter(MultiPage<T> page, FragmentManager fragmentManager) : base(fragmentManager, FragmentStatePagerAdapter.BehaviorResumeOnlyCurrentFragment)
+#pragma warning restore CS0618 // Type or member is obsolete
 		{
 			_page = page;
 			_fragmentManager = fragmentManager;

--- a/Xamarin.Forms.Platform.Android/EntryAccessibilityDelegate.cs
+++ b/Xamarin.Forms.Platform.Android/EntryAccessibilityDelegate.cs
@@ -1,5 +1,5 @@
 ﻿using Android.Views.Accessibility;
-using AndroidX.Core.View.Accessibiity;
+using AndroidX.Core.View.Accessibility;
 using Xamarin.Forms.Platform.Android.FastRenderers;
 
 namespace Xamarin.Forms.Platform.Android

--- a/Xamarin.Forms.Platform.Android/Renderers/ShellContentFragment.cs
+++ b/Xamarin.Forms.Platform.Android/Renderers/ShellContentFragment.cs
@@ -15,12 +15,33 @@ namespace Xamarin.Forms.Platform.Android
 {
 	public class ShellContentFragment : Fragment, AndroidAnimation.IAnimationListener, IShellObservableFragment, IAppearanceObserver
 	{
+		// AndroidX.Fragment packaged stopped calling CreateAnimation for every call
+		// of creating a fragment
+		bool _isAnimating = false;
+
 		#region IAnimationListener
 
 		void AndroidAnimation.IAnimationListener.OnAnimationEnd(AndroidAnimation animation)
 		{
 			View?.SetLayerType(LayerType.None, null);
 			AnimationFinished?.Invoke(this, EventArgs.Empty);
+			_isAnimating = false;
+		}
+
+		public override void OnResume()
+		{
+			base.OnResume();
+			if (!_isAnimating)
+			{
+				View?.SetLayerType(LayerType.None, null);
+				AnimationFinished?.Invoke(this, EventArgs.Empty);
+			}
+		}
+
+
+		public override void OnViewStateRestored(Bundle savedInstanceState)
+		{
+			base.OnViewStateRestored(savedInstanceState);
 		}
 
 		void AndroidAnimation.IAnimationListener.OnAnimationRepeat(AndroidAnimation animation)
@@ -75,6 +96,7 @@ namespace Xamarin.Forms.Platform.Android
 		public override AndroidAnimation OnCreateAnimation(int transit, bool enter, int nextAnim)
 		{
 			var result = base.OnCreateAnimation(transit, enter, nextAnim);
+			_isAnimating = true;
 
 			if (result == null && nextAnim != 0)
 			{

--- a/Xamarin.Forms.Platform.Android/Renderers/ShellFragmentPagerAdapter.cs
+++ b/Xamarin.Forms.Platform.Android/Renderers/ShellFragmentPagerAdapter.cs
@@ -6,13 +6,17 @@ using Xamarin.Forms.Platform.Android.AppCompat;
 
 namespace Xamarin.Forms.Platform.Android
 {
+#pragma warning disable CS0618 // Type or member is obsolete
 	internal class ShellFragmentPagerAdapter : FragmentPagerAdapter
+#pragma warning restore CS0618 // Type or member is obsolete
 	{
 		bool _disposed;
 		ShellSection _shellSection;
 		IShellSectionController SectionController => (IShellSectionController)_shellSection;
 
+#pragma warning disable CS0618 // Type or member is obsolete
 		public ShellFragmentPagerAdapter(ShellSection shellSection, FragmentManager fragmentManager) : base(fragmentManager, FragmentStatePagerAdapter.BehaviorResumeOnlyCurrentFragment)
+#pragma warning restore CS0618 // Type or member is obsolete
 		{
 			_shellSection = shellSection;
 			SectionController.ItemsCollectionChanged += OnItemsCollectionChanged;

--- a/Xamarin.Forms.Platform.Android/Renderers/ShellRenderer.cs
+++ b/Xamarin.Forms.Platform.Android/Renderers/ShellRenderer.cs
@@ -268,7 +268,7 @@ namespace Xamarin.Forms.Platform.Android
 			FragmentTransaction transaction = manager.BeginTransactionEx();
 
 			if (animate)
-				transaction.SetTransitionEx((int)global::Android.App.FragmentTransit.EnterMask);
+				transaction.SetTransitionEx((int)global::Android.App.FragmentTransit.FragmentOpen);
 
 			transaction.ReplaceEx(_frameLayout.Id, fragment);
 			transaction.CommitAllowingStateLossEx();

--- a/Xamarin.Forms.Platform.Android/Xamarin.Forms.Platform.Android.csproj
+++ b/Xamarin.Forms.Platform.Android/Xamarin.Forms.Platform.Android.csproj
@@ -45,7 +45,7 @@
   <ItemGroup>
     <PackageReference Include="Xamarin.AndroidX.Lifecycle.LiveData" Version="2.2.0.4" />
     <PackageReference Include="Xamarin.Google.Android.Material" Version="1.2.1.1" />
-    <PackageReference Include="Xamarin.AndroidX.Legacy.Support.V4" Version="1.0.0.6" />
+    <PackageReference Include="Xamarin.AndroidX.Legacy.Support.V4" Version="1.0.0.7" />
   </ItemGroup>
   <ItemGroup>
     <ProjectReference Include="..\Xamarin.Forms.Core\Xamarin.Forms.Core.csproj">

